react/content-list.jsx

```diff
@@ -36,7 +42,14 @@ let ContentList = React.createClass({
 
         contentList.push({key, address, addressString, title, name, label, body, comments})
       }
-      this.setState({contentList})
+
+      this.setState({contentList, waitOnUserSetup})
+
+      if (waitOnUserSetup) {
+        setTimeout(function () {
+          this.setState({waitOnUserSetup: false})
+        }.bind(this), 2000)
```

Comment:
This seems a little odd - why two seconds? Surely there is some other way to check this other than a delay? e.g., "If no content & user has not set up yet, do not display 'no content' message".